roles/openldap/tasks/install-configure.yml

```diff
@@ -0,0 +1,80 @@
+- name: install the openldap packages
+  apt: name={{ item }} state=installed
+  with_items: "{{ openldap.ldap_pkgs }}"
+
+- name: create directory for ldif files
+  file: path={{ openldap.config_dir }}
+        state=directory
+        owner=root
+        group=root
+        mode=0400
+
+- name: copy templates
+  template: src={{ item }}
+            dest={{ openldap.config_dir }}/{{ item | basename }}
```

Comment:
Why are these needing basenames? The template file should be named the same as the destination file.